walkeeper/src/timeline.rs

```diff
@@ -166,16 +166,16 @@ impl SharedState {
 
 /// Database instance (tenant)
 pub struct Timeline {
-    pub timeline_id: ZTimelineId,
+    pub zttid: ZTenantTimelineId,
     mutex: Mutex<SharedState>,
     /// conditional variable used to notify wal senders
     cond: Condvar,
 }
 
 impl Timeline {
-    fn new(timelineid: ZTimelineId, shared_state: SharedState) -> Timeline {
+    fn new(zttid: &ZTenantTimelineId, shared_state: SharedState) -> Timeline {
```

Comment:
if we copy it anyway we can accept owned value directly. IMO it is better to accept a reference when there is no need to copy, i e argument type indicates function demands

Reply:
ok